tests/integration/records/unload-test.js

```diff
@@ -1488,3 +1488,8 @@ test('many:many async unload', function (assert) {
     assert.equal(person1Friends.isDestroyed, true, 'previous ManyArray is destroyed in the runloop after refetching');
   });
 });
+
+test('1 sync : 1 async');
```

Comment:
Fwiw I do t think QUnit supports this syntax (assuming you are intending to basically create a skip?)...

Reply:
@rwjblue creating a todo, not a skip.